### ISSUE TITLE
The result of trained net is different from the p_estimate.mat / trained by helm.h5 file.

### ISSUE THREAD

**Issue by zzziuuu** [justinYoon12/OceanPINN#1]
The results I obtained by using your data and network differ significantly from those in your paper. I would like to ask for your guidance on how to make the necessary adjustments. 
I noticed that the parameter.json file is not included in the code you uploaded, then I have conducted experiments using the existing data.json file, I am not sure the reason of the question.

**Comment by zzziuuu** [comment]
![1w2w12w120w](https://github.com/user-attachments/assets/3255e6ac-858e-4bd3-9025-689306a9c1fe)

**Comment by justinYoon12** [comment]
I think the example case may have been uploaded incorrectly. The PDE coefficient in the data.json file is too small compared to the appropriate value for training. I've added parameter.json; could you please run it again with this parameter? L=1e4-1e5 is appropriate.

**Comment by Tianyima247** [comment]
Hello, Professor. I have a question to ask you. When I set the parameters according to the "parameter" and simulate the training and prediction results at SNR = 12dB, there is a big difference from those in the paper. I don't know what the reason is.Maybe the coefficient of the loss function is inappropriate Could you please take a look at the results? Thank you!

<img width="1461" height="1113" alt="Image" src="https://github.com/user-attachments/assets/bcb8d092-94db-4d5d-b95a-c22e7d8f7dfe" />

<img width="1681" height="746" alt="Image" src="https://github.com/user-attachments/assets/2a46b5f1-7a57-4525-be8c-a5dc983761c1" />

<img width="1683" height="751" alt="Image" src="https://github.com/user-attachments/assets/271dcb37-b60a-4675-bd44-5db00c52d4a8" />
